### PR TITLE
Auth token, separate routes file, rate limiting, cors parenthesis, "@types/node": "^15.14.1"

### PR DESCRIPTION
**Pull request by dohitdeegwal**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,13 +18,14 @@
   "dependencies": {
     "@prisma/client": "^4.15.0",
     "@types/cors": "^2.8.13",
-    "@types/node": "^20.3.0",
+    "@types/node": "^15.14.1",
```
Comment (ggsingla) [inline package.json:21]: Why do we need degrade the version?

File: routes.ts
Comment (ggsingla) [inline routes.ts:1]: Can we put this file inside the routes folder and rename it to main.ts

File: package.json (hunk 1 above)
Comment (dohitdeegwal) [reply package.json:21]: Necessary for Prisma client to work. @Adityagupta1625 suggested this solution.

File: routes.ts
Comment (dohitdeegwal) [reply routes.ts:1]: Done

**Review by pixelperfectnitj — APPROVED** [review]

**Pull request merged**